plugins/activity/plugin.kdl

```diff
@@ -2,12 +2,14 @@ publisher "mitre"
 name "activity"
 version "0.1.0"
 license "Apache-2.0"
+
 entrypoint {
-    on arch="aarch64-apple-darwin" "./target/debug/activity"
-    on arch="x86_64-apple-darwin" "./target/debug/activity"
-    on arch="x86_64-unknown-linux-gnu" "./target/debug/activity"
-    on arch="x86_64-pc-windows-msvc" "./target/debug/activity.exe"
+  on arch="aarch64-apple-darwin" "activity"
+  on arch="x86_64-apple-darwin" "activity"
+  on arch="x86_64-unknown-linux-gnu" "activity"
+  on arch="x86_64-pc-windows-msvc" "activity.exe"
```

Comment:
The modification to the path is necessary for `download-manifest`-based plugin installs, but breaks the system we put in place for local plugin testing, as the plugin resolver relies on this path to know where to grab the locally-built plugin.

I think we do make the change, but we'll need to come up with a solution for supporting both ASAP